### DISCUSSION TITLE
get contents of plain text page

### DISCUSSION THREAD

**Discussion by RobertFlatt** [r0x0r/pywebview#1238] [Q&A] [answered]
A pywebview window contains only plain text.

How do I read this text?

I understand I should do this on an `events.loaded`, but not how to use `get_elements()` to get the contents.

**Comment by RobertFlatt** [comment, marked as answer]
Answered my own question  `get_elements('body')`

**Reply by r0x0r** [reply]
Note that the next version will feature DOM support, so getting plain text of the body will be either `window.dom.body.text` or `window.dom.get_element('body').text`.

**Comment by 79102576** [comment]
get_elements()获取的列表是个字典,其中有包含text和html的key,
